src/get-started/flutter-for/ios-devs.md

```diff
@@ -1280,9 +2013,9 @@ For further details on internationalization and localization in Flutter,
 see the [internationalization guide][], which has sample code
 with and without the `intl` package.
 
-### What is the equivalent of CocoaPods? How do I add dependencies?
+### Managing dependencies
 
-In iOS, you add dependencies by adding to your `Podfile`.
+In iOS, you add dependencies with Cocoapods by adding to your `Podfile`.
```

Comment:
```suggestion
In iOS, you add dependencies with CocoaPods by adding to your `Podfile`.
```
Maybe you can also do a global search for "Cocoapods" (should be CocoaPods) and "Github" (should be GitHub)?

Reply:
Okay sure